deploy_timemap/templates/nginx.conf.j2

```diff
@@ -7,4 +7,7 @@ server {
 	location / {
 		return 403;
 	}
+
+	# enter certbot block below
```

Comment:
wow this is also a much nicer way of inserting into a particular place. I've noticed that the tasks in 'nginx.yaml' sometimes incorrectly insert lines at the end of the configuration- I'll add an issue to update them to work like this!

Reply:
Sweet! Yeah, the config file is really tricky as it feels pretty fragile if it's already there.